.github/workflows/check_data_leaks.yml

```diff
@@ -0,0 +1,21 @@
+name: Check Data Leaks
+
+on: [pull_request]
+
+defaults:
+  run:
+    working-directory: .
+
+jobs:
+  scan-data-leaks:
+    name: gitleaks
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+
+      - name: detect-secrets
+        run : |
+          pip install detect-secrets
+          detect-secrets scan .
```

Comment:
We could also enable GitHub Advanced Security’s secret scanner. Will this be a useful way of finding data leaks in our parsers?

Reply:
Removed, we are going to try [Github Secret Scanner](https://docs.github.com/en/code-security/getting-started/securing-your-repository#configuring-secret-scanning).